app/com/yetu/oauth2provider/oauth2/handlers/AuthorizationHandler.scala

```diff
@@ -106,43 +99,27 @@ class AuthorizationHandler(authAccessService: IAuthCodeAccessTokenService,
     Future.successful(loggedIn)
   }
 
-  def getStoredAccessToken(authInfo: AuthInformation) = {
+  def getStoredAccessToken(authInfo: AuthInfo[YetuUser]) = {
     logger.warn("...getStoredAccessToken :: NOT_IMPLEMENTED")
```

Comment:
Can we see where this method is used and see if we can implement it, now that we have some persistence? This however does the reverse lookup.